src/sentry/web/frontend/organization_event.py

```diff
@@ -0,0 +1,26 @@
+from __future__ import absolute_import
+
+from django.http import HttpResponseRedirect, Http404
+from django.core.urlresolvers import reverse
+
+from sentry.models import Event
+
+
+def event_redirect(request, organization_slug, client_event_id):
+    """
+    Given a client event id and organization slug, redirects to the event page
+    """
+    try:
+        event = Event.objects.select_related('project').get(event_id=client_event_id)
```

Comment:
i don't think you can do this query because there is no index on `event_id` -- it is indexed with project so it will time out / not complete in production. that's why this had to be reverted: https://github.com/getsentry/sentry/pull/10035 https://github.com/getsentry/sentry/pull/10266

cc @mattrobenolt to confirm

Reply:
Absolutely cannot do this. It will result in a full sequence scan. And even if it did return, it'd likely return more than 1 row since event_ids aren't globally unique. They are only unique within a project.